ingestion/src/metadata/cli/docker.py

```diff
@@ -9,18 +9,97 @@
 import click
 import requests as requests
 
+from metadata.ingestion.ometa.ometa_api import OpenMetadata
+from metadata.ingestion.ometa.openmetadata_rest import MetadataServerConfig
+
 logger = logging.getLogger(__name__)
-calc_gb = 1024 * 1024 * 1000
+calc_gb = 1024 * 1024 * 1024
 min_memory_limit = 6 * calc_gb
 
 
+def start_docker(docker, start_time, file_path):
+    logger.info("Running docker compose for Open Metadata....")
+    if file_path:
+        docker.compose.up(detach=True, build=True)
+    else:
+        docker.compose.up(detach=True)
+
+    logger.info(
+        "Docker Compose for Open Metadata successful. Waiting for ingestion to complete...."
+    )
+    metadata_config = MetadataServerConfig.parse_obj(
+        {
+            "api_endpoint": "http://localhost:8585/api",
+            "auth_provider_type": "no-auth",
+        }
+    )
+    ometa_client = OpenMetadata(metadata_config).client
+    while True:
+        try:
+            ometa_client.get(f"/tables/name/bigquery_gcp.shopify.dim_customer")
```

Comment:
It might be cleaner to use `metadata.get_by_name` instead of accessing the client directly. If we change to that, we might need to add a time.sleep if the response is None, as the request won't fail